src/runtime/virtcontainers/sandbox.go

```diff
@@ -1234,9 +1234,24 @@ func (s *Sandbox) startVM(ctx context.Context) (err error) {
 		return err
 	}
 
+	if prestartHookFunc != nil {
+		hid, err := s.GetHypervisorPid()
+		if err != nil {
+			return err
+		}
+		s.Logger().Infof("hypervisor pid is %v", hid)
+		ctx = context.WithValue(ctx, "hypervisor-pid", hid)
```

Comment:
static checks are complaining about:
```
Error: sandbox.go:1243:32: SA1029: should not use built-in type string as key for value; define your own type to avoid collisions (staticcheck)
		ctx = context.WithValue(ctx, "hypervisor-pid", hid)
		                             ^
make: *** [Makefile:41: static-checks] Error 1
Error: Process completed with exit code 2.
```